app/controllers/Contributions.scala

```diff
@@ -57,6 +57,7 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
       val intCmp = request.getQueryString("INTCMP")
       val refererPageviewId = request.getQueryString("REFPVID")
       val refererUrl = request.headers.get("referer")
+      val platform = request.getQueryString("platform")
```

Comment:
Those both sound like good improvements, thanks. I'll set the project up properly on my own machine to ensure everything builds, I was just being lazy before, not wanting to get AWS credentials and so on.